libopae/plugins/xfpga/sysfs.c

```diff
@@ -139,6 +140,14 @@ static sysfs_fpga_device _devices[SYSFS_MAX_DEVICES];
 		}                                                              \
 	} while (0);
```

Comment:
nit - removing the ; at the end forces it to appear where the macro is expanded.  As written, it could be omitted.